test/replication-status.test.js

```diff
@@ -58,7 +58,7 @@ Object.keys(testAPIs).forEach(API => {
 
     it('has correct replication info after load', async () => {
       await db.add('hello')
-      await db.close()
+      // await db.close()
```

Comment:
Same comment as for https://github.com/orbitdb/orbit-db/pull/677/files#r325600547